system-test/spanner.ts

```diff
@@ -8894,6 +8894,24 @@ describe('Spanner', () => {
         mismatchedColumnError(done, DATABASE, googleSqlTable);
       });
 
+      it('GOOGLE_STANDARD_SQL should use getTransaction for executing sql', async () => {
+        const promise = await DATABASE.getTransaction({optimisticLock: true});
+        const transaction = promise[0];
+
+        try {
+          const [rows] = await transaction!.run('SELECT * FROM TxnTable');
+          assert.strictEqual(rows.length, googleSqlRecords.length);
+        } catch (err) {
+          if ((err as grpc.ServiceError).code === grpc.status.ABORTED) {
+            assert.ok(err, 'Transaction is aborted');
+          } else {
+            console.log('ERROR: ', err);
```

Comment:
Shouldn't this also fail the test? I don't think we expect any errors here.

Reply:
Updated